eve/main.yml

```diff
@@ -158,7 +158,7 @@ stages:
       path: eve/workers/doc-builder.yaml
       images:
         doc-builder:
```

Comment:
here you can set the following `doc-builder: docs` as the context and the Dockerfile are in the same directory